src/commands/staff/whois.ts

```diff
@@ -0,0 +1,44 @@
+import { staffPerms } from '@lib/permissions';
+import { userParser } from '@root/src/lib/arguments';
+import { GuildMember, Message, MessageEmbed } from 'discord.js';
+import prettyMilliseconds from 'pretty-ms';
+
+export const description = 'Gives an overview of a member\'s info.';
+export const usage = '<user>';
+export const runInDM = false;
+export const aliases = ['member'];
+
+export function permissions(msg: Message): boolean {
+	return staffPerms(msg);
+}
+
+export function run(msg: Message, member: GuildMember): Promise<Message> {
+	const roles = member.roles.cache.map(role => role.name).sort().join(', ');
```

Comment:
```suggestion
	const roles = member.roles.cache.size > 1
		? member.roles.cache.filter(r => r.id !== r.guild.id).sort().array().join(' ')
		: 'none';
```
Removes the `@everyone` role and shows as mentions for *fun colors*